### PR TITLE
Updating the Documentation

### PR DESCRIPTION
**Pull request by emarinier**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,135 @@
...
+
+| sample  | fastq_1                    | fastq_2                    | assembly                     |
+| ------- | -------------------------- | -------------------------- | ---------------------------- |
+| SAMPLE1 | /path/to/sample1_fastq1.fq | /path/to/sample1_fastq2.fq | /path/to/sample1_assembly.fa |
+| SAMPLE2 | /path/to/sample2_fastq1.fq |                            |                              |
+
+The columns are defined as follows:
+
+- `sample`: The unique sample identifier to associate with the reads (and optionally the assembly).
+- `fastq_1`: A URI (ex: a file path or web address) to either single-end FASTQ-formatted reads or one pair of pair-end FASTQ-formatted reads.
```
Comment (apetkau) [inline README.md:19]: Is single-end analysis supported right now? If not, I would leave out mentioning single-end analysis.

File: README.md [hunk 2]
```diff
@@ -2,7 +2,135 @@
...
+- `fastq_2`: (Optional) If `fastq_1` is paired-end, then this field is a URI to reads that are the other pair of reads associated with `fastq_1`.
+- `assembly`: (Optional) A URI to an assembly associated with the sample, so that it may be referenced on the command line by the sample identifier for use as the reference for the whole pipeline. However, it may be easier to leave these fields blank and specify the reference using the `--refgenome` parameter.
+
+The structure of this file is defined in [assets/schema_input.json](assets/schema_input.json). Please see [assets/samplesheet.csv](assets/samplesheet.csv) to see an example of a samplesheet for this pipeline.
+
+# Parameters
+
+## Mandatory
+
+The mandatory parameters are as follows:
```
Comment (apetkau) [inline README.md:29]: I'd mention it's mandatory to have one of `--refgenome` or `--reference_sample_id`

File: README.md [hunk 3]
```diff
@@ -2,7 +2,135 @@
...
+- `--reference_sample_id`: the sample identifier of a sample in the samplesheet that contains a provided `assembly` to use as a reference genome during pipeline analysis
+
+Please use only one of `--refgenome` or `--reference_sample_id` and not both.
+
+### SNVPhyl Parameters
+
+- `--window_size`: The window size for determining whether a region is high density.
+- `--density_threshold`: The minimum number of SNVs within the window size for a region to be considered high density.
+- `--min_coverage_depth`: The minimum depth of coverage for a position within the genome to pass the mapping quality check and be included in the analysis.
+- `--min_mapping_percent_cov`: The total percentage of positions within the genome that must have a depth of coverage greater than the minimum depth of coverage specified in order to pass the mapping quality check and be included in the analysis.
```
Comment (apetkau) [inline README.md:50]: I'd leave out the **and be included in the analysis**. This is used more to flag problematic samples in the mapping quality check/output file. It has no impact on if samples get included in the analysis.

File: README.md [hunk 4]
```diff
@@ -2,7 +2,135 @@
...
+- `--min_mean_mapping_quality`: The minimum mean mapping quality score for all reads in a pileup to be included in the analysis.
+- `--snv_abundance_ratio`: The proportion of reads required to support a variant to be included in the analysis.
+- `--min_repeat_length`: The minimum length when identifying repeats.
```
Comment (apetkau) [inline README.md:53]: Change to *identifying repeats __on the reference genome__*.

File: README.md [hunk 5]
```diff
@@ -2,7 +2,135 @@
...
+- `--min_repeat_pid`: The minimum percent identity when identifying repeats.
+- `--skip_density_filter`: Whether or not to skip filtering low SNV density regions.
+
+Please refer to the [SNVPhyl documentation](https://snvphyl.readthedocs.io/en/latest/) for more detailed information about pipeline parameters.
+
+# Running
+
+## Test Data
```
Comment (apetkau) [inline README.md:61]: I would also mention you can use `nextflow run phac-nml/snvphylnfc -profile singularity,test --outdir results` for running the test profile.

File: README.md [hunk 6]
```diff
@@ -2,7 +2,135 @@
...
+
+In order to run the pipeline with provided data, please run:
+
+```
+nextflow run main.nf -profile singularity --input https://raw.githubusercontent.com/phac-nml/snvphylnfc/dev/assets/samplesheet.csv --refgenome https://raw.githubusercontent.com/phac-nml/snvphylnfc/main/assets/reference.fasta --outdir results
```
Comment (apetkau) [inline README.md:66]: Use `phac-nml/snvphylnfc` instead of `main.nf` for how to run pipeline.

File: README.md [hunk 7]
```diff
@@ -2,7 +2,135 @@
...
+- `vcf2snv/snvTable.tsv`: a table of all detected variant sites
+- `vcf2snv/vcf2core.tsv`: a table of the evaluated core positions in each reference fasta sequence
+- `vcf2snv/snvAlignment.phy`: an alignment of SNVs used to generate the phylogenetic tree
+- `verifying/mappingQuality.txt`: describes how well the given reads mapped to the reference genome
+
+For more detailed information, please refer to the [SNVPhyl Documentation](https://snvphyl.readthedocs.io/en/latest/user/output/).
+
+## IRIDA Next Integration File
+
+A JSON file for loading the data into IRIDA Next is output by this pipeline. The format of this JSON file is specified in our [Pipeline Standards for the IRIDA Next JSON](https://github.com/phac-nml/pipeline-standards#32-irida-next-json). This JSON file is written directly within the `--outdir` provided to the pipeline with the name `irida.output.json.gz` (ex: `[outdir]/irida.output.json.gz`).
```
Comment (apetkau) [inline README.md:90]: Name **iridanext.output.json.gz**

File: docs/usage.md [hunk 8]
```diff
@@ -38,10 +38,10 @@ An [example samplesheet](../assets/samplesheet.csv) has been provided with the p
 The typical command for running the pipeline is as follows:
 
 ```bash
-nextflow run main.nf --input ./samplesheet.csv --outdir ./results -profile singularity
+nextflow run main.nf -profile singularity --input https://raw.githubusercontent.com/phac-nml/snvphylnfc/dev/assets/samplesheet.csv --refgenome https://raw.githubusercontent.com/phac-nml/snvphylnfc/main/assets/reference.fasta --outdir results
```
Comment (apetkau) [inline docs/usage.md:41]: use `nextflow run phac-nml/snvphylnfc`

**Review by apetkau — CHANGES_REQUESTED** [review]
This looks great. Thanks so much Eric 😄 

In addition to comments:

* Please update `CITATIONS.md` file with the SNVPhyl citations found inhttps://snvphyl.readthedocs.io/en/latest/citation/
    * Also include reference to Jill Hagey's repo in citations (https://github.com/DHQP/SNVPhyl_Nextflow)
* In main README include acknowledgement of nf-core as described in https://nf-co.re/docs/contributing/tutorials/unofficial_pipelines#acknowledging-nf-core

File: README.md (hunk 1 above)
Comment (emarinier) [reply README.md:19]: Yup, I added single-end a few PRs back. I can't remember which one, but it works.

File: docs/usage.md (hunk 8 above)
Comment (emarinier) [reply docs/usage.md]: Updated in 887e53de4eb3cbbd83f6cb0d881e4bbd1e26e931

File: README.md (hunk 7 above)
Comment (emarinier) [reply README.md]: Updated in d47cead498a1fe3ccccfc06977f4a9a19f97afd2

File: README.md (hunk 6 above)
Comment (emarinier) [reply README.md]: Updated in f9d87fcfb004ec10792bdce136ab155ccba31fd3

File: README.md (hunk 5 above)
Comment (emarinier) [reply README.md:68]: Updated in a114e651bc85254cc30ff1c79174611979206d8c

File: README.md (hunk 4 above)
Comment (emarinier) [reply README.md]: Updated in 2be4bd2038d29cb4e55e43ed46e76c0d65f6b9b4

File: README.md (hunk 3 above)
Comment (emarinier) [reply README.md]: Updated in f634f0209546ea4c89a85904cc07f1d79a7823d0

File: README.md (hunk 2 above)
Comment (emarinier) [reply README.md:29]: Updated in 19be15205924c6e8522987df6402c7a50e907e78

**Comment by emarinier** [comment]
> This looks great. Thanks so much Eric 😄
> 
> In addition to comments:
> 
>     * Please update `CITATIONS.md` file with the SNVPhyl citations found inhttps://snvphyl.readthedocs.io/en/latest/citation/
>       
>       * Also include reference to Jill Hagey's repo in citations (https://github.com/DHQP/SNVPhyl_Nextflow)
> 
>     * In main README include acknowledgement of nf-core as described in https://nf-co.re/docs/contributing/tutorials/unofficial_pipelines#acknowledging-nf-core

Updated in 9aec0993b84e14a93f2f2eab6792ad3298f0a880

**Review by apetkau — APPROVED** [review]
Amazing 🥳 . Thanks so much :). Approved ✔️

**Pull request merged**